subsystems/chassis/SolenoidShifters.java

```diff
@@ -62,25 +57,22 @@ public SolenoidShifters(int portUp, int portDown) {
 	/**
 	 * A subsystem for managing a solenoid for a shifting drivetrain.
 	 *
-	 * @param module
-	 *        The ID of the PCM for the double solenoid
-	 * @param portUp
-	 *        The first port of the double solenoid
-	 * @param portDown
-	 *        The second port of the double solenoid
+	 * @param module   The ID of the PCM for the double solenoid
+	 * @param portUp   The first port of the double solenoid
+	 * @param portDown The second port of the double solenoid
 	 */
 	public SolenoidShifters(int module, int portUp, int portDown) {
 		this(new DoubleSolenoid(module, portUp, portDown), false);
 	}
 
 	@Override
-	protected void initDefaultCommand() {
-		setDefaultCommand(new ChassisShift(this, SolenoidShifters.ShiftState.DOWN));
+	public Command getDefaultCommand() {
```

Comment:
according to the wpilib devs, `getDefaultCommand()` is not meant to be overridden. Please delete this method and add a note to the javadoc for this class that the normal default command is `ChassisShift(this, SolenoidShifters.ShiftState.DOWN)` an is set with `subsystemname..setDefaultCommand(...)`